Documentation/developer-guide.md

```diff
@@ -30,61 +27,48 @@ In order to build .NET Command Line Interface, you need the following installed
 
 ## Building/Running
 
-1. Run `build.cmd` or `build.sh` from the root depending on your OS. If you don't want to execute tests, run `build.cmd -Targets Prepare,Compile` or `./build.sh --targets Prepare,Compile`.
-2. Use `artifacts/{os}-{arch}/stage2/dotnet` to try out the `dotnet` command. You can also add `artifacts/{os}-{arch}/stage2` to the PATH if you want to run `dotnet` from anywhere.
+1. Run `build.cmd` or `build.sh` from the root depending on your OS. If you don't want to execute tests, run `build.cmd /t:Compile` or `./build.sh /t:Compile`. 
+2. Use `artifacts/{RID}/stage2/dotnet` to try out the `dotnet` command. You can also add `artifacts/{os}-{arch}/stage2` to the PATH if you want to run `dotnet` from anywhere.
 
 ## A simple test
 
-1. `cd TestAssets\TestProjects\TestSimpleIncrementalApp`
-2. `dotnet restore`
-3. `dotnet run`
+1. `cd {new directory}`
+2. `dotnet new`
+3. `dotnet restore3`
+4. `dotnet run3`
 
 ## Running tests
 
-All the CLI tests are located under `test`. In order to run them, after doing a restore on the CLI repo just do the following:
-
-1. Navigate to a test project, for instance: `cd test\dotnet-test.UnitTests`
-2. `dotnet test`
+1. To run all tests invoke `build.cmd` or `build.sh` which will build the product and run the tests.
+2. To run a specific test, cd into that test's directory and execute `dotnet test`
```

Comment:
I think we should mention that for the CLI tests itself, it expects stage2 to be on the path and the NUGET_PACKAGES should be set to <repo_root>/.nuget/packages